### ISSUE TITLE
A few contacts in the Contacts app not synced fully to Dialer

### ISSUE THREAD

**Issue by glocalglocal** [SimpleMobileTools/Simple-Dialer#312]
When I replaced the stock contact/dialer with the Simple Tools equivalents ages ago, a small number of contacts ended up mangled (eg 2-3 separate entries for the same contact, some of them containing only the photo or a gmail address). 

After some cleaning up in Contacts, the problem is that a few entries in Contacts and Dialer were never synced. Eg I am looking at two entries in Contacts. They both have multiple numbers, street address, photo and email. The only difference is that one is a Favourite, but otherwise there is nothing apparently special about them. 

In Dialer, they appear on the respective tabs as expected (Contact and Favourite tabs with their photo) but if I open them from the Contacts tab in Dialer to see their content, all I see is the photo and my gmail account where they were stored previously. Even their names are missing. If I switch to the Favourites tab in Dialer and select the one Favourite contact, Dialer will behave as normal (ie it shows a menu with the available numbers from the Contacts app and I can select one etc.). Because the second contact is not a Favourite, there is no way I can call it from Dialer, but I can from the Contacts app. 

To sum up, certain few contacts in Contacts lose most of their details in Dialer when Dialer was first installed. The only info that survived is the name and photo on the list, the photo when I open the contact, and the favourite status. Dialer will still call such a contact but only from the Favourite tab if the contact is a favourite. 

Simple Dialer v5.11.4, Simple Contacts Pro 6.17.1, Android 7.

**Comment by tibbi** [comment]
What contact source are you saving the contact to? I mean the last field at the contact details screen. Does it say your email, Phone, SIM or something like that?

**Comment by JulienFavre** [comment]
I have a similar problem.
I'm just using Simple Dialer 5.11.5 with the Android Contacts app, I have Android 9. There is no synchronization with Google and the contacts are nor on the SIM card. It's just the contacts on the phone.
One of my contact which is also a favorite is not showing in Simple Dialer. It's not showing in the main list nor in the favorites.
I've notice that Simple Dialer does not show contacts without a phone number, I think it's ok because it's a dialer after all !
The problem is that the favorite contact that is not showing has a phone number. In fact it has 3 numbers.
The only difference I see between this contact and others is that it has a note field.

**Comment by JulienFavre** [comment]
I found what the issue was, at least for me.
When a contact has a number which is the same as another contact's number, only the first contact in the list is displayed.
I removed the landline numbers from the contact which was not showing because another contact had these same numbers. I left only the unique mobile number and now this contact shows up !
